index.html

```diff
@@ -2,11 +2,13 @@
 <html lang="en">
 
 <head>
```

Comment:
Please capitalize the commits so that they look clean and meaningful. Kindly read [this](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) article and write professional commit messages.